www/docs/sidebars.js

```diff
@@ -14,119 +14,145 @@ module.exports = {
     {
       type: "doc",
       id: "homepage",
-      label: "Overview",
+      label: "Introduction",
     },
     {
       type: "doc",
       id: "quickstart/quick-start",
-      label: "Quickstart",
+      label: "Quick Start Guide",
```

Comment:
Can we keep quickstart in one word? That's at least the way I see it styled in most places :)

Reply:
I changed it back to one word